webdriver/tests/support/fixtures.py

```diff
@@ -19,7 +19,7 @@
 PAGE_LOAD_TIMEOUT = 3
 IMPLICIT_WAIT_TIMEOUT = 0
 
-# The webdriver session can outlive a pytest session
+# The webdriver session can outlive a pytest session.
```

Comment:
Note that this change will be reverted before landing. It was used to trigger all the wdspec tests which are the only consumers of websockets in wpt.